.env

```diff
@@ -1,7 +1,8 @@
-EXTERNAL_SERVER_HOST=127.0.0.1
-EXTERNAL_SERVER_PORT=47124
-INTERNAL_SERVER_HOST=127.0.0.1
-INTERNAL_SERVER_PORT=47126
+EXTERNAL_HOST=127.0.0.1
+EXTERNAL_PORT=47124
+EXTERNAL_FQDN=x.bitcoinx.io
```

Comment:
This needs to be 127.0.01 in the repo, otherwise it breaks things?

Reply:
Okay. Thanks

Reply:
It looks like you pushed the wrong file for the public repo.

Reply:
Aha... I've pushed more updates... The reading from the .env file was happening after the internal_host, external_fqdn and external_host had already been set... Have now moved it to happen earlier.

Reply:
I don't understand. This is still the same. What does it have to do with regtest?